src/extension/uilities.js

```diff
@@ -0,0 +1,24 @@
+// The utilities files to store functions that would be used in mutiple components.
+import parse from 'html-react-parser'
+
+// functional components to highlight search text and covert indentifers to the links
+// same function from iSamples_results.js
+export function ResultWrapper(props){
```

Comment:
Please make sure you run the formatter on this file, there should be a space between the close parentheses and curly brace.